kratos/sources/variables.cpp

```diff
@@ -816,7 +821,12 @@ void KratosApplication::RegisterVariables() {
     //--------------- Meshing ApplicationApplication -------------------//
 
     KRATOS_REGISTER_VARIABLE(NODAL_ERROR)
-    KRATOS_REGISTER_VARIABLE(NODAL_ERROR_COMPONENTS)
+    KRATOS_REGISTER_3D_VARIABLE_WITH_COMPONENTS(NODAL_ERROR_COMPONENTS)
+    KRATOS_REGISTER_VARIABLE(ELEMENT_ERROR )
+    KRATOS_REGISTER_VARIABLE(ELEMENT_H )
+    KRATOS_REGISTER_VARIABLE(RECOVERED_STRESS )
+    KRATOS_REGISTER_VARIABLE(ERROR_INTEGRATION_POINT )
+    KRATOS_REGISTER_VARIABLE(CONTACT_PRESSURE )
```

Comment:
this does not exist yet?
Arent the DEM guys using sth like this?

Reply:
![imagen](https://user-images.githubusercontent.com/19991680/42572180-6c49b49e-8519-11e8-97b4-27e59503937b.png)
Nope